source/packages/services/events-processor/src/api/targets/target.dao.spec.ts

```diff
@@ -63,10 +63,7 @@ describe('TargetDao', () => {
                 pk: `S:${stubbedGoodEmailItem.subscriptionId}`,
                 sk: `ST:email:${stubbedGoodEmailItem.address}`,
                 gsi2Key: `ES:${eventSourceId}:${principal}:${principalValue}`,
-                gsi2Sort: `S:${stubbedGoodEmailItem.subscriptionId}:ST:email:${stubbedGoodEmailItem.address}`,
```

Comment:
Can you provide explanation on why are these removed form the unit tests ?

Reply:
@tsugunao still need your input on the question

Reply:
@hassankhokhar I am not get involved with this change. I guess you meant @komushi

Reply:
@hassankhokhar 
The purpose of this fix is to remove those redundant attributes. Those attributes will not be used by other logics - including events-alerts.
After removing those attributes the UT code needs to match the specification.
This above is the reason I removed those attributes.